django_simple_bulma/finders.py

```diff
@@ -37,24 +37,23 @@ def _get_bulma_css(self):
         scss_string = ""
         for var, value in self.variables.items():
             scss_string += f"${var}: {value};\n"
-
+        
+        # SASS wants paths with forward slash:
+        sass_bulma_path = str(self.simple_bulma_path).replace('\\', '/')
         # Now load bulma
-        sass_path = self.simple_bulma_path / 'bulma.sass'
-        scss_string += f'@import "{sass_path}";'
+        scss_string += f'@import "{sass_bulma_path}/bulma.sass";'
 
         # Now load in the extensions that the user wants
         if self.extensions == "_all":
-            extensions_path = self.simple_bulma_path / "sass" / "extensions" / "_all"
-            scss_string += f'@import "{extensions_path}";\n'
+            scss_string += f'@import "{sass_bulma_path}/sass/extensions/_all";\n'
         elif isinstance(self.extensions, list):
             for extension in self.extensions:
 
                 # Check if the extension exists
-                extensions_folder = self.simple_bulma_path / "sass" / "extensions"
+                extensions_folder = sass_bulma_path / "sass" / "extensions"
```

Comment:
this line seems like it would crash, since you've typecast this into a string on line 42 and strings don't support the `/` operator like this, only Paths do. Did you test this code?

Reply:
I tested it just by running it on the project I'm using, which doesn't guarantee everything is tested :)